### PR TITLE
Feature/205/kansas smittys config

### PR DESCRIPTION
**Pull request by antoineeudes**

### CODE REVIEW

File: src/components/context/ExperienceContext.tsx [hunk 1]
```diff
@@ -109,19 +110,19 @@ export default ({
...
-        .collection(`experiences/${experienceName}/reactions`)
+        .collection(`experiences/${venueName}/reactions`)
         .add(newReaction);
     },
-    [firebase, experienceName]
+    [firebase, venueName]
   );
 
-  const store = { reactions, addReaction };
+  const store = { reactions, addReaction, venueName };
```
Comment (mathieud) [inline src/components/context/ExperienceContext.tsx:125]: Why not add a name to the Venue, that's the same value as the id? I don't think we should have this information split.

File: src/components/venues/Jazzbar/JazzTab/JazzTab.tsx [hunk 2]
```diff
@@ -344,7 +345,7 @@ const Jazz: React.FunctionComponent<PropsType> = ({ setUserList }) => {
               width="100%"
               height="100%"
               className="youtube-video"
-              src="https://www.youtube.com/embed/dqZAA8ZIAVE?autoplay=1"
+              src={venue.iframeUrl ? `${venue.iframeUrl}?autoplay=1` : ""}
```
Comment (mathieud) [inline src/components/venues/Jazzbar/JazzTab/JazzTab.tsx:348]: `iframeUrl` should not be optional

File: src/components/venues/Jazzbar/JazzTab/JazzTab.tsx [hunk 3]
```diff
@@ -406,11 +407,13 @@ const Jazz: React.FunctionComponent<PropsType> = ({ setUserList }) => {
                 isVideoFocused ? "col-5" : "col-12"
               } table-container`}
             >
-              <TableHeader
-                seatedAtTable={seatedAtTable}
-                setSeatedAtTable={setSeatedAtTable}
-                experienceName={experience.associatedRoom}
-              />
+              {experienceContext && (
```
Comment (mathieud) [inline src/components/venues/Jazzbar/JazzTab/JazzTab.tsx:410]: Why do we need to check this all the time now? Can't see what changed.

File: src/components/context/ExperienceContext.tsx (hunk 1 above)
Comment (thecadams) [reply src/components/context/ExperienceContext.tsx:125]: Yep and we can use the document ID as the venue name. That will ensure it’s unique

Comment (thecadams) [reply src/components/context/ExperienceContext.tsx:125]: Yep and we can use the document ID as the venue name. That will ensure it’s unique

**Pull request merged**